pages/workspaces/[workspaceId]/index.tsx

```diff
@@ -50,6 +54,8 @@ interface WorkspaceDashboardProps {
 }
 
 const WorkspaceDashboard = ({ workspace }: WorkspaceDashboardProps) => {
+  const [showWelcome, setShowWelcome] = useLocalStorage("show-welcome", true);
```

Comment:
I worry about relying on local storage too much since we've talked about doing continous app-cache busting as we move pretty fast on new features in the API etc:

- #2794 

Could we rely on the `is_onboarded` bool on the user object from the api instead? There are downsides to that too: we'd be overloading the onboarding abit

Reply:
That should work! Local storage was the suggested solution, but I'm not opposed to keeping it in the API.

Reply:
I could be swayed either way: not a super strong opinion. If Nick thought that using local storage was the way to go, we can always refactor in the future 🫡